src/menu/cutsceneview.c

```diff
@@ -0,0 +1,67 @@
+/*
+ * This software is licensed under the terms of the MIT License.
+ * See COPYING for further information.
+ * ---
+ * Copyright (c) 2011-2019, Lukas Weber <laochailan@web.de>.
+ * Copyright (c) 2012-2019, Andrei Alexeyev <akari@taisei-project.org>.
+ */
+
+#include "eventloop/eventloop.h"
+#include "menu/menu.h"
+#include "taisei.h"
+
+#include "cutsceneview.h"
+#include "common.h"
+#include "options.h"
+#include "global.h"
+#include "video.h"
+#include "cutscenes/cutscene.h"
+#include "cutscenes/scenes.h"
+#include "progress.h"
+
+static void draw_cutsceneview_menu(MenuData *m) {
+	draw_options_menu_bg(m);
+	draw_menu_title(m, "Cutscene Viewer");
+	draw_menu_list(m, 100, 100, NULL, SCREEN_H);
+}
+
+static void restart_menu_bgm(CallChainResult ccr) {
+	audio_bgm_play(res_bgm("menu"), true, 0, 0);
+}
+
+static void cutscene_player(MenuData *m, void *arg) {
+	CutsceneID *cutscene_id = arg;
+	assume(cutscene_id != NULL);
+
+	CallChainResult ccr;
+	cutscene_enter(CALLCHAIN(restart_menu_bgm, ccr.ctx), *cutscene_id);
+}
+
+MenuData* create_cutsceneview_menu(void) {
```

Comment:
Please always use identifier-aligned pointer asterisks rather than type-aligned ones for new code; it's even desirable to correct it in neighboring old code.